src/hooks/useTheme.ts

```diff
@@ -1,45 +1,39 @@
 import { useEffect } from "react";
 import { useDispatch, useSelector } from "react-redux";
-import { Theme } from "../models";
 
 import { actions, selectUI } from "../stores/ui";
 
 export const useTheme = () => {
   const dispatch = useDispatch();
-  const { theme } = useSelector(selectUI);
+  let { theme } = useSelector(selectUI);
```

Comment:
Are you sure about that? const should work

Reply:
I reassigned it in line 18-22.